src/client/js/controller/welcome.js

```diff
@@ -63,32 +63,29 @@ class Welcome {
         this.view.help.setWelcomeContent(
           'FunCoNN (Beta)',
           [
-            '<b>Welcome to FunCoNN (Beta): Functional Connectivity on NemaNode</b>',
+            'Welcome to <b>FunCoNN (Beta): <u>Fun</u>ctional <u>Co</u>nnectivity on <u>N</u>ema <u>N</u>ode</b>',
             '<p>',
-            'This is an interactive browser of functional connectivity measurements of',
-            '<i>C. elegans</i> overlaid on select connectomics data using the software package',
-            '<b>NemaNode</b>.',
+            'This is an interactive browser of functional connectivity measurements of <i>C. elegans</i> overlaid',
+            'on select connectomics data using the <a href="https://nemanode.org/" target="_blank">Nema Node</a> software.',
             '<p>',
-            'Functional connectivity measurements show the casual direction of information',
-            'flow, polarity and functional strength of neural connections based on',
-            'optogenetic perturbations and calcium imaging.',
+            'Functional connectivity measurements show the causal direction of information flow, polarity',
+            'and functional strength of neural connections based on optogenetic perturbations and calcium',
+            'imaging. Functional measurements shown in this visualization are from a forthcoming',
+            'manuscript from the <a href="http://leiferlab.princeton.edu/" target="_blank">Leifer Lab</a>: Randi, Sharma, Dvali and Leifer, &#8220Direct measurements',
```

Comment:
This is totally fine, but I always try and avoid content/language that is guaranteed to go stale (c.i.p. "forthcoming manuscript"). I might insert a comment like `<!-- manuscript update here -->` and/or create an issue with a link to this line of code. Or... do they have a doi yet? You can just say "manuscript" and link to it...

Reply:
...I realize that the language is probably not yours Ben.